translator/writer/writer.h

```diff
@@ -89,6 +89,20 @@ class UsdArnoldWriter {
                 _scope = _scope.substr(0, _scope.length() - 1);            
         }
         
+    }
+    const std::string &GetMtlScope() const {return _mtlScope;}
+    void SetMtlScope(const std::string &scope) {
+        _mtlScope = scope;
+        if (!_mtlScope.empty()) { 
+            // First character needs to be a slash
+            if (_mtlScope[0] != '/')
+                _mtlScope = std::string("/") + _mtlScope;
+            // Last character should *not* be slash, otherwise we could have 
+            // double slashes in the nodes names, which can crash usd
+            if (_mtlScope.back() == '/')
```

Comment:
it the name has multiple slashes in the end (hopefully it shouldn't happen, but it can), this will remove only one, should we try to remove all the slashes ?
I was thinking that we could may be use SdfPath instead, in the future, as they are efficient memory wise, and that would avoid us to write checking code ?